data/core/terrain-graphics/new-macros.cfg

```diff
@@ -2360,7 +2360,14 @@ X*#endarg
     {NEW:VILLAGE {TERRAINLIST} {IMAGESTEM} (ANIM=[01~03]) (TIME={TIME})}
 #enddef
 
-#define NEW:VILLAGE_TOD_P TERRAINLIST PROB IMAGESTEM
+#define NEW:VILLAGE_TOD TERRAINLIST IMAGESTEM
+
+#arg PROB
+25#endarg
+
+#arg TIME
+first_watch,dusk,dusk1,dusk2,short_dark,long_dark1,underground#endarg
```

Comment:
Hmm, this is really less than ideal since it won't trigger for any custom ToD, and yet I can't think of a better way to do it (basing it on the ToD bonus would mean the lights stay on during second watch, for example). If only we could base it on _next_ turn's ToD bonus...

Reply:
Also, I wonder if it might be better to make this an argument to the macro instead of hard-coded - you could then rename this macro DEFAULT_TIMES or something and pass it to all the default terrains.

Reply:
Oh yeah, this means underground villages are permanently lit up? That's also less than ideal... but again I don't have a solution for it...

Reply:
The default times are my best guess at what would look good, but someone can change them in UMC with a custom terrain if they want; the macro will accept different TIME arguments.